.changeset/sixty-houses-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+'@celo/celocli': minor
```

Comment:
ive been thinking that viem changes are a patch since they should just be an internal thing and are not a new feature for users. thoughts.?

Reply:
yeah that was a mistake